parsely_raw_data/pixel_logs/__init__.py

```diff
@@ -1,19 +0,0 @@
-__license__ = """
-Copyright 2016 Parsely, Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-"""
-from .log_line import LogLine, VisitorInfo, TimestampInfo, SessionInfo, DisplayInfo
-
-__all__ = ["LogLine", "VisitorInfo", "TimestampInfo", "SessionInfo",
-           "DisplayInfo"]
```

Comment:
I killed the sub-package because it seemed unnecessary for a single module.  There's just `event.py` and `event.thrift` now.